tensorflow/python/kernel_tests/self_adjoint_eig_op_test.py

```diff
@@ -150,13 +150,14 @@ def Test(self):
 
 
 if __name__ == '__main__':
-  for dtype in np.float32, np.float64:
+  for dtype in np.float32, np.float64, np.complex64, np.complex128:
     for size in 1, 2, 5, 10:
       for batch_dims in [(), (3,)] + [(3, 2)] * (max(size, size) < 10):
         shape = batch_dims + (size, size)
         name = '%s_%s' % (dtype.__name__, '_'.join(map(str, shape)))
         setattr(SelfAdjointEigTest, 'testSelfAdjointEig_' + name,
                 _GetSelfAdjointEigTest(dtype, shape))
-        setattr(SelfAdjointEigGradTest, 'testSelfAdjointEigGrad_' + name,
-                _GetSelfAdjointEigGradTest(dtype, shape))
+        if dtype is [np.float32, np.float64]:
```

Comment:
Please fix is -> in. It would be preferable to test all gradients. You might need to fix the gradient computation for complex types. We've had a few bugs like that. I'm guessing that the transpose here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/linalg_grad.py#L215

should be a conjugate transpose, but I'm not sure.